src/main/kotlin/io/sip3/salto/ce/sip/SipCallHandler.kt

```diff
@@ -245,10 +245,12 @@ open class SipCallHandler : AbstractVerticle() {
     open fun calculateByeTransactionMetrics(transaction: SipTransaction) {
         val createdAt = transaction.createdAt
 
-        val attributes = excludeSessionAttributes(transaction.attributes).apply {
-            transaction.srcAddr.host?.let { put("src_host", it) }
-            transaction.dstAddr.host?.let { put("dst_host", it) }
-        }
+        val attributes = excludeSessionAttributes(transaction.attributes)
```

Comment:
In `excludeSessionAttributes` we already have a transformantion to MutableMap. I'm curious if it might be better to change the order in all methods but to change `excludeSessionAttributes` implementation. Like:
```
transaction.attributes
    .toMetricsAttributes()
    .excludeSessionAttributes()
```

Reply:
We can merge this 2 methods:

```
transaction.attributes
    .toMetricsAttributes(SESSION_ATTRIBUTES)
    ...

fun Map<String, Any>. toMetricsAttributes(exclude: List<String> = emptyList()): MutableMap<String, Any> {
...
```